lib/libc/minimal/include/time.h

```diff
@@ -43,6 +46,11 @@ typedef _TIME_T_ time_t;
 typedef _SUSECONDS_T_ suseconds_t;
 #endif
 
+#if !defined(__clock_t_defined)
```

Comment:
Any particular reason for this typedef dance? Are you expecting something to define this type differently somehow?

Reply:
Just copy-pasting :-)

Reply:
yeah, maybe someday I'll understand why C libraries do this kind of thing.